art/api/tests/test_asset_make_model.py

```diff
@@ -0,0 +1,19 @@
+from django.test import TestCase
+from ..models import MakeModel
+
+
+class MakeModelTestCase(TestCase):
+    def setUp(self):
+        MakeModel.objects.create(make_label="Microsoft Lifechat LX-6000")
+        self.laptop_make = MakeModel.objects.get (make_label="Microsoft Lifechat LX-6000")
+
+    def test_can_create_asset_make(self):
+        self.assertEqual(self.laptop_make.make_label, 'Microsoft Lifechat LX-6000')
```

Comment:
we should also create a new object here and check whether the count in the db changes